README.md

```diff
@@ -13,7 +13,7 @@ This collection contains event source plugins, event filters and example ruleboo
 
 * ansible-rulebook >= 1.0.0
 * python >= 3.9
-* ansible >= 2.13
+* ansible-core >= 2.14
```

Comment:
Maybe I am wrong, but ansible-core 2.14 has not reached EOL?

Reply:
Right, i will update this in a separate change.